example/gardener-local/gardenlet/values.yaml

```diff
@@ -78,6 +78,8 @@ config:
           secretRef:
             name: internal-domain-internal-local-gardener-cloud
             namespace: garden
+      extensions:
+      - type: local-ext-seed
```

Comment:
Why is this added?

Reply:
To activate the extension for the seed in the local setup.